### PR TITLE
docs: add example for awscc_gamelift_fleet

### PR DESCRIPTION
**Pull request by novekm**
**IMPORTANT:** This resource creates a GameLift Fleet which can take some time to fully activate. This can lead to a long-running (>25m) `terraform apply` and is normal. In the example, I added a link to the [AWS Docs](https://docs.aws.amazon.com/gamelift/latest/developerguide/fleets-creating-debug.html) for troubleshooting potential misconfigurations that could cause the resource to fail successful activation. 

### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

Relates OR Closes #0000

### CODE REVIEW

File: examples/resources/awscc_gamelift_fleet/gamelift_fleet.tf [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["gamelift.amazonaws.com"]
+    }
+  }
+}
+
```
Comment (wellsiau-aws) [inline examples/resources/awscc_gamelift_fleet/gamelift_fleet.tf:65]:
```suggestion
```

remove blank space

File: examples/resources/awscc_gamelift_fleet/gamelift_fleet.tf [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      parameters = "yourParameterKey:yourParameterValue yourParameterKey2:YourParameterValue2"
+    },
+    {
+      concurrent_executions = 1
+      launch_path           = "/local/game/path-to-your-file"
+      parameters = "yourParameterKey:yourParameterValue yourParameterKey2:YourParameterValue2"
+    },
+    ]
+  }
+}
```
Comment (wellsiau-aws) [inline examples/resources/awscc_gamelift_fleet/gamelift_fleet.tf:22]:
```suggestion
resource "awscc_gamelift_fleet" "example" {
  name              = "ExampleFleet"
  build_id          = awscc_gamelift_build.example.id
  compute_type      = "EC2"
  ec2_instance_type = "c5.large"
  fleet_type        = "ON_DEMAND"

  runtime_configuration = {
    server_processes = [
    {
      concurrent_executions = 1
      launch_path           = "/local/game/path-to-your-file"
      parameters = "yourParameterKey:yourParameterValue yourParameterKey2:YourParameterValue2"
    },
    {
      concurrent_executions = 1
      launch_path           = "/local/game/path-to-your-file"
      parameters = "yourParameterKey:yourParameterValue yourParameterKey2:YourParameterValue2"
    },
    ]
  }
}
```

small formatting

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]
@novekm , looks good, just a small nit pick to tidy up the doc

File: templates/resources/gamelift_fleet.md.tmpl [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+# {{.Name}} ({{.Type}})
+
+{{ .Description | trimspace }}
+
+## Example Usage
+
+**IMPORTANT:** The S3 permissions listed in the example are meant for testing purposes only. Restrict these permissions further for enhanced security. Amazon GameLift Fleets can take some time to finish activating. This can lead to a long-running (>25min) `terraform apply` and is normal. If you encounter errors with your fleet not activating successfully, check the troubleshooting steps listed in [this doc](https://docs.aws.amazon.com/gamelift/latest/developerguide/fleets-creating-debug.html).
+
+
```
[the remark this answers is not in the document]
Comment (novekm) [reply templates/resources/gamelift_fleet.md.tmpl:16]: updated 👍🏾

File: examples/resources/awscc_gamelift_fleet/gamelift_fleet.tf (hunk 1 above)
Comment (novekm) [reply examples/resources/awscc_gamelift_fleet/gamelift_fleet.tf:65]: updated 👍🏾

File: examples/resources/awscc_gamelift_fleet/gamelift_fleet.tf (hunk 2 above)
Comment (novekm) [reply examples/resources/awscc_gamelift_fleet/gamelift_fleet.tf]: updated 👍🏾

**Review by wellsiau-aws — APPROVED** [review]
LGTM

**Review by wellsiau-aws — APPROVED** [review]
LGTM, thanks!

**Pull request merged**